d2l-dom.js

```diff
@@ -6,53 +6,44 @@ import {
 } from '@brightspace-ui/core/helpers/dom.js';
 
 window.D2L = window.D2L || {};
+window.D2L.Dom = window.D2L.Dom || {};
+window.D2L.Dom.findComposedAncestor = findComposedAncestor;
+window.D2L.Dom.getComposedChildren = getComposedChildren;
+window.D2L.Dom.getComposedParent = getComposedParent;
+window.D2L.Dom.isComposedAncestor = isComposedAncestor;
+window.D2L.Dom.getOffsetParent = function(node) {
+	if (!window.ShadowRoot) {
+		return node.offsetParent;
+	}
 
-/** @polymerBehavior */
```

Comment:
I think this used to be important for linting, but I don't remember the details...

Reply:
Yeah none of the other things in here have it, so I figured I'd roll the dice and kill it haha. These aren't "real" Polymer behaviours anyway.